doc/porting_guide.md

```diff
@@ -390,51 +415,53 @@ Let's assume the new platform's name is np2000. And an early advise: as a rule o
 
-#### Using a non-Make build system
+#### Using a Third Party IDE (non-make)
 
 In the case that the _make_ build system is unavailable for your platform you can migrate the 
-Xively C Client build environment to your platform's own development environment.
-While we cannot completely predict how this process would work for every IDE and toolchain, here are some 
-suggested steps to follow when working through this process:
+Xively C Client build environment to your platform's own Integrated Development Environment (IDE).
 
-- import all of the Xively C Client source files in the src/libxively directory. This is where all of the Xively Client's platform independent code resides.
-- import all of the source files in one of the subdirectories of src/bsp/platform.  
+While we cannot completely predict how this process would work for every IDE and toolchain, here are some suggested steps to follow when working through this process:
+
+- import all of the Xively C Client source files in the `src/libxively` directory into your project workspace. This is where all of the Xively Client's platform independent code resides. 
+	- Some of these files might not be required depending on your desired library configuration.  You could prune some files later by comparing the fileset against a mac / linux build log.
+- import all of the source files in one of the subdirectories of `src/bsp/platform`.  
   The Xively C Client BSPs contain all of the missing hooks which tie the platform independent 
   code to a particular device. Complete implementations should include a source file for each
   of the BSP subsystems (networking, memory, time, etc).
 	- NOTE: The Xively C Client contains reference BSP implementations for POSIX and CC3200. We also have partial implementations for specific networking APIs. 
-    Modules from these 'incomplete' BSP implementations could be used as substitutes. For instance, on devices that mirror POSIX completely except for networking, like Simplelink, the networking module from src/bsp/platform/posix could be ovewritten with the one from src/bsp/platform/simplelink_incomplete. 
-- import one of the BSP TLS implementations in src/bsp/tls.  Currently we provide two different TLS BSP implementations: WolfSSL or mbedTLS.
-- add all of the directories in src/libxively to your include path
-- add the /include directory to your include path
+    Modules from these 'incomplete' BSP implementations could be used as substitutes. For instance, on devices that mirror POSIX completely except for networking, like Simplelink, the networking module from `src/bsp/platform/posix` could be ovewritten with the one from `src/bsp/platform/simplelink_incomplete`. 
+- import one of the BSP TLS implementations in `src/bsp/tls`.  Currently we provide two different TLS BSP implementations: WolfSSL or mbedTLS.
+- add all of the directories in `src/libxively` to your include path
```

Comment:
first bullet point is to import the source files, this bullet point is to add them to the include path.